linode/manager
# M3-5204: Add Cost Estimator for Custom Images

## Description

Adds price estimation for creating custom images from existing Linode Disks. The price estimation is based on the size of the disk and the current unit cost of $0.10 / GB of Disk storage. Included is also utility function to convert between storage units easily and another utility function to calculate total prices given a unit price and a given quantity.

## How to test

Navigate to the CreateCustomImageTab component and select a Linode and a Disk. The cost estimator should show the correct price for the size of the disk.

**How do I run relevant unit tests?**

`yarn test unitPriceCalculator`
`yarn test unitConversions --t convertStorageUnit`
